src/test/java/com/vmtapp/enterprise/VarmentTmsApplicationTests.java

```diff
@@ -33,7 +33,8 @@
     @Test
     void contextLoads() {
     }
-  
+
+    // John is a client -> Test if John has access to tickets -> John is not permitted
```

Comment:
Clarified test method specifics in comment.

Reply:
If putting a comment above a method, make it proper JavaDoc.